internal/platform/platform.go

```diff
@@ -88,7 +88,7 @@ func SelectManifest(ctx context.Context, src content.ReadOnlyStorage, root ocisp
 				return m, nil
 			}
 		}
-		return ocispec.Descriptor{}, errdef.ErrNotFound
+		return ocispec.Descriptor{}, fmt.Errorf("%s: no matching manifest was found in the manifest list", root.Digest)
```

Comment:
Could you wrap `errdef.ErrNotFound` using `%w` in `fmt.Errorf`? So that invokers can check the general error using `errors.Is()`?